test/over_react/component/pure_component_mixin_test.dart

```diff
@@ -0,0 +1,129 @@
+// Copyright 2020 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+@TestOn('browser')
+library pure_ui_component_test;
+
+import 'package:over_react_test/over_react_test.dart';
+import 'package:test/test.dart';
+
+import './fixtures/pure_test_components.dart';
+import '../../test_util/test_util.dart';
+
+/// Main entry point for `PureComponentMixin` testing
+main() {
+  group('PureComponentMixin', () {
+    const initialChildren = ['initial'];
```

Comment:
Looks like we're only testing prop updates here. Should we also be testing the effect of state updates/non-updates?

Reply:
Probably not a huge deal since the implementation is the same as for props 🤷

Reply:
There is a test at the very beginning that uses `forceUpdate`

Reply:
Ah, I see it now 👍